hack/examples/overlays/clusterclasses/nutanix/kustomization.yaml.tmpl

```diff
@@ -72,7 +72,3 @@ patches:
       kind: KubeadmConfigTemplate
     path: ../../../patches/cis/kubeadmconfigtemplate/kubelet-file-permissions.yaml
   # END CIS patches
-
-  - target:
-      kind: NutanixMachineTemplate
-    path: ../../../patches/nutanix/remove-cluster-and-subnet-from-machinetemplates.yaml
```

Comment:
Why is this being removed? It also doesn't result in any changes in the example manifests though right?